bellows/cli/util.py

```diff
@@ -112,9 +113,9 @@ def cfg(config_id, value):
 
 
 @asyncio.coroutine
-def setup_application(dev, database_file, startup=True):
+def setup_application(dev, database_file, startup=True, baudrate=None):
     s = bellows.ezsp.EZSP()
-    yield from s.connect(dev)
+    yield from s.connect(dev, baudrate=baudrate)
```

Comment:
Can you change either this call or the call in setup() above so that they use the same style.